cmd/standalone/Dockerfile

```diff
@@ -20,5 +20,6 @@ EXPOSE 16686
 
 COPY ./jaeger-ui-build /go/src/jaeger-ui-build
 COPY ./cmd/standalone/standalone-linux /go/bin/
+COPY ./cmd/standalone/sampling_strategies.json /go/src/config/
 
-CMD ["/go/bin/standalone-linux","--query.static-files=/go/src/jaeger-ui-build/build/"]
+CMD ["/go/bin/standalone-linux","--query.static-files=/go/src/jaeger-ui-build/build/","--sampling.strategies-file=/go/src/config/sampling_strategies.json"]
```

Comment:
The standalone will be provided with a sampling strategies file which defaults the sampling probability to 1

Reply:
clever